mrchecker-framework-modules/mrchecker-core-module/src/main/java/com/capgemini/mrchecker/test/core/utils/encryption/view/DataEncryptionGUI.java

```diff
@@ -0,0 +1,67 @@
+package com.capgemini.mrchecker.test.core.utils.encryption.view;
+
+import javax.swing.*;
+
+import com.capgemini.mrchecker.test.core.utils.encryption.CryptParams;
+import com.capgemini.mrchecker.test.core.utils.encryption.controller.DataEncryptionController;
+import com.capgemini.mrchecker.test.core.utils.encryption.controller.DataEncryptionControllerImp;
+
+public class DataEncryptionGUI extends JFrame implements IView {
+	
+	private final DataEncryptionController dataEncryptionController;
+	
+	private JPanel contentPane;
+	private JTabbedPane mainPane;
+	private JPanel encryptionPane;
+	private JPanel decryptionPane;
+	private JTextField encryptionTextField;
+	private JTextField encryptionKeyTextField;
+	private JButton chooseEncryptionKeyButton;
+	private JButton encryptButton;
+	private JLabel encryptionKeyLabel;
+	private JLabel encryptionLabel;
+	private JLabel encryptionResultLabel;
+	private JTextField encryptionResultTextField;
+	private JTextField decryptionTextField;
+	private JTextField decryptionResultTextField;
+	private JButton decryptButton;
+	private JButton chooseDecryptionKeyButton;
+	private JTextField decryptionKeyTextField;
+	private JLabel decryptionKeyLabel;
+	private JLabel decryptionLabel;
+	private JLabel decryptionResultLabel;
+	
+	public DataEncryptionGUI() {
+		dataEncryptionController = new DataEncryptionControllerImp(this);
```

Comment:
This cannot be implemented like that. We need another way to instantiate and pass DataEncryptionController implementation.